resources/styles.sdk4.css

```diff
@@ -19,7 +19,9 @@
   font-family: Colfax-Regular;
   text-anchor: start;
   text-length: 64;
-  fill: #FFFFFF
+  fill: #FFFFFF;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Can you please change the styles.sdk5.css as well?

Reply:
Oh right! 🤦🏻‍♂️ 😁

Reply:
Done